relnotes/verify.feature.md

```diff
@@ -0,0 +1,4 @@
+* `ocean.core.Verify`
+
+  New module with a single function, `verify`, intended to serve as drop-in
+  replacement for `assert` to comply to Sociomantic assert/enforce policies.
```

Comment:
I would explain the reasons for it a bit more, or at least link to the relevant issue where it is explained.